docs/modules/ROOT/pages/getting-started/relationships-and-advanced-filtering.adoc

```diff
@@ -45,53 +50,49 @@ const personNode = new Cypher.Node({
     labels: ["Person"],
 });
 ```
++
+So far, the relationship is not connected to the nodes. 
+This is because here you are creating a variable to hold the relationship matched by the pattern, not the relationship itself.
 
-Note how, so far the relationship is not connected to the Nodes, this is because what you are creating here is a variable to hold the relationship matched by the pattern, not the relationship itself.
-
-=== Creating the pattern
-
-You now can define an arbitrarily complex pattern with `new Cypher.Pattern` using the variables already declared:
-
+. Now use `new Cypher.Pattern` to define an arbitrarily complex pattern with the variables already declared:
++
 ```javascript
 const pattern = new Cypher.Pattern(movieNode).related(actedIn).to(personNode);
 ```
++
+Note that, in Cypher Builder, patterns work similarly to Cypher, with each element of the chain being either a relationship or a node.
 
-The patterns in Cypher Builder work similarly to Cypher, with each element of the chain being either a relationship or a node.
-
-Now, you can switch the `movieNode` variable for the `pattern` in the Match clause:
-
+. Switch the `movieNode` variable for the `pattern` in the `MATCH` clause:
++
 ```javascript
 const clause = new Cypher.Match(pattern)
 ```
 
-Executing it should yield the following Cypher:
-
+. Run the script.
+You should get the following Cypher:
++ 
 ```cypher
 MATCH (this0:Movie)-[this1:ACTED_IN]->(this2:Person)
 WHERE (this0.title = $param0 AND this0.released < $param1)
 RETURN this0.title, this0.tagline, this0.released
 ```
-
++
 ```javascript
 { param0: 'The Matrix', param1: 2000 }
 ```
 
-Note how, despite changing the Match target, all the filters and relationships are still pointing towards the correct variable: `this0`
-
-However, the pattern is slightly off. Instead of a **Person** acting in a **Movie** there is a **Movie** acting on a **Person**!
+=== Cardinality, direction, and properties
 
-Just a small tweak in the pattern direction should fix this:
+Despite changing the `MATCH` target in the previous steps, all the filters and relationships featured in the query are still pointing towards the correct variable: `this0`. 
+However, the pattern is incorrect -- instead of returning a `Person` acting in a `Movie`, there is a `Movie` acting on a `Person`.
+This is how you fix that:
```

Comment:
Maybe we can point to the exact addition that needs to be made here?

Something like: `To change that, add `.withDirection("left")` to the pattern